src/main/webapp/script.js

```diff
@@ -182,3 +186,59 @@ function loadSelector(selector, filename) {
     $(selector).load(filename);
   });
 }
+
+/**
+ * Loads the stored values for the sorting selector and populates dashboard with
+ * the Entertainment Items.
+ */
+function loadDashboard() {
+  $('#sortingDirection').change(function() {
+    localStorage.setItem('sortDir', $(this).val());
+    getDashboardItems();
+  });
+
+  $(document).ready(function() {
+    const sortDir = localStorage.getItem('sortDir');
+
+    if (sortDir !== null) {
+      $(`#sortingDirection`).val(sortDir);
+    }
+
+    getDashboardItems();
+  });
+}
+
+/**
+ * Updates the display of buttons used for pagination if the
+ * current page is full.
+ *
+ * @param { number } itemsInPage - number of items loaded in the page
+ * @param { string } pageCursor - opaque key representing the cursor for the
+ *     next page
+ */
+function updatePagination(itemsInPage, pageCursor) {
+  const MAX_ITEMS_PER_PAGE = 18;
+  // Display pagination button only when needed(page is full).
+  if (itemsInPage === MAX_ITEMS_PER_PAGE) {
+    $('#paginationNav').removeClass('d-none');
+    $('#nextLink').attr('href', '/index.html?cursor=' + pageCursor);
+  } else {
+    $('#paginationNav').addClass('d-none');
+  }
+}
+
+/**
+ * Finds a query string parameter in the current Url.
+ *
+ * @param { string } param - the parameter to look for in the window Url
+ * @returns { string } if the parameter exists it returns the value found,
+ *     otherwise an empty string
+ */
+function getUrlParam(param) {
```

Comment:
This function is similar to Eric's  getItemId() in itempagescript.js. Lets consolidate them

Reply:
I deleted getItemId() and instead added the use of a more generic function getUrlParam() in last commit.